src/Database.pas

```diff
@@ -2911,7 +2913,7 @@ procedure CreateUnit(p, mix: integer);
       Status := 0;
       SavedStatus := 0;
       Inc(Model[mix].Built);
-      Home := -1;
+      Home := 0; //Lagi: -1 is homeless, support free settler
```

Comment:
You're right about Despotism, I forgot about the surrounding code block (`if` statement).

Sorry for not being clear enough about `freeciv`. I mentioned freeciv as a theoretical alternative, not that I propose to change rules that way.

In fact, I'm just being very conservative about game rules changes in general.
In fact, I didn't even include this change: https://gitlab.com/vn971/cevo/commit/6e81c57eddca2495c9a565956ba95b0ea6ae6cd3
It has a bad description, but it basically fixes an original C-evo bug with not the cheapest unit being used for defense if many have the same defense strength, and not the closest city tile chosen above those with similar resources.
I don't know if/how/when it should be fixed, but this would break the tutorial Example game where some tactics are being shown.
Steffen also has some magic "gamever" in the code, maybe you can have alternative rules for different maps. Then again, if a new C-evo version will come out, it will be incompatible in both ways. I'm careful about changes..